lib/ansible/modules/cloud/vmware/vsphere_copy.py

```diff
@@ -58,6 +58,12 @@
         set to C(no) when no other option exists.
     default: 'yes'
     type: bool
+  timeout:
+    description:
+      - The timeout in seconds for the upload to the datastore.
+    default: 10
```

Comment:
Could you please increase time more to something like 60 or so ? Rest LGTM

Reply:
@Akasurde 
I could do that, but that would change the behavior compared to previous versions. If no timeout is specified, it defaults to 10 seconds (reference: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py#L1148)

So, it would seem safer to me to keep the default for the parameter on 10 seconds, that's what it has always been and nobody has complained until now.

However, if you prefer we set it to 60 now, I can live with that. Just give the word and I will make the change.